config/config/sc-config.yaml

```diff
@@ -1061,20 +1061,28 @@ networkPolicies:
 #     ingress: {}
 #     egress: {}
 # ---
-  harbor:
-    enabled: true
-    chartmuseum:
-      # IP to S3
-      egressIPs:
+  global:
+    apiserver:
+      ips:
+        - "0.0.0.0/0"
+      port: 6443
+    scNodes:
+      ips:
+        - "0.0.0.0/0"
+    wcIngress:
+      ips:
         - "0.0.0.0/0"
-    core:
-      # IP to S3, dex
-      egressIPs:
+    objectStorage:
+      ips:
         - "0.0.0.0/0"
+      port: 443
```

Comment:
Didn't we decide to use `ports` as a list for all endpoints that we don't control? Same thing for some more below.

Reply:
Hmm, I don't recall us talking about a list of ports anytime

Reply:
Maybe we did and I just misunderstood, but I don't think that a list of port makes sense.

Reply:
Maybe you're right. I'm just thinking that there could be corner cases where it's needed, e.g. if we were using object storage in several regions/clouds that have different ports.
So I agree that we might never have to use it, but could be good to have.